### PR TITLE
RMSNorm epsilon refine in the example

### PR DESCRIPTION
**Pull request by pengxin99**
According to the [RMSNorm define](https://docs.pytorch.org/docs/stable/generated/torch.nn.modules.normalization.RMSNorm.html) in Pytorch, the epsilon operation is in the rsqrt()

## Summary by CodeRabbit

* **Bug Fixes**
  * Improved numerical stability in normalization calculations by adjusting epsilon placement within the normalization formula.

* **Tests**
  * Updated normalization tests to reflect improved stability calculations.

### CODE REVIEW

**Review by LeiWang1999 — APPROVED** [review]

**Comment by LeiWang1999** [comment]
LGTM, Thanks, also cc @chengyupku

**Comment by LeiWang1999** [comment]
@codex review
